lib/sinon/behavior.js

```diff
@@ -8,6 +8,7 @@ var valueToString = require("@sinonjs/commons").valueToString;
 var exportAsyncBehaviors = require("./util/core/export-async-behaviors");
 
 var concat = arrayProto.concat;
+var flat = arrayProto.flat;
```

Comment:
We are strictly using ES5.1, predating the existence of this method by half a decade. Thanks for keeping a reference to the original, though :-)

Reply:
This is why all tests fail in Node 6, 8 and 10 (as you can see by clicking the failed build), as neither has that method in the runtime.

Reply:
Resolved in 8e00c22.  Using slice instead, which is better anyway.